packages/webview_flutter/webview_flutter_wkwebview/lib/src/webview_cupertino.dart

```diff
@@ -45,5 +45,6 @@ class CupertinoWebView implements WebViewPlatform {
   }
 
   @override
-  Future<bool> clearCookies() => MethodChannelWebViewPlatform.clearCookies();
+  Future<bool> clearCookies() =>
+      WebViewCookieManagerPlatform.instance!.clearCookies();
```

Comment:
Should include a `null` check. The bang operator (`!`) should only be used if it is clear in the same code block that the field is set.